src/probability/hitting_time.lean

```diff
@@ -0,0 +1,217 @@
+/-
+Copyright (c) 2022 Kexing Ying. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Kexing Ying, Rémy Degenne
+-/
+import probability.stopping
+
+/-!
+# Hitting time
+
+Given a stochastic process, the hitting time provides the first time the process ``hits'' some
+subset of the state space. The hitting time is a stopping time in the case that the time index is
+discrete and the process is adapted.
```

Comment:
This is true much more generally. You could write that this is the case for which we proved it.
See for example "the measurability of hitting times" by Richard E. Bass (and the follow up paper "correction to 'the measurability of hitting times'").